File: beispiele-einzeln/klasse-und-objekt/klasse-und-objekt.py
class Car:
    # Konstukteur der Klasse
    def __init__(self, color, model):
        self.color = color
        self.model = model

    # Method
    def drive(self):
        print(f"The {self.color} {self.model} is driving.")

# Instanz der Klasse
my_car = Car("red", "Toyota")
my_car.drive()

# Hier beschreibt die Klasse Car allgemeine Eigenschaften und Verhaltensweisen für Autoobjekte. 
# Bei der Erstellung eines Objekts können wir bestimmte Werte für die Eigenschaften festlegen.

File: beispiele-einzeln/klasse-und-objekt/klasse-und-objekt.java
class Car {
    // Eigenschaften
    String color;
    String model;

    // Konstrukteur der Klasse
    public Car(String color, String model) {
        this.color = color;
        this.model = model;
    }

    // Method
    public void drive() {
        System.out.println("The " + color + " " + model + " is driving.");
    }

    public static void main(String[] args) {
        Car myCar = new Car("red", "Toyota");
        myCar.drive();
    }
}

// Hier beschreibt die Klasse Car allgemeine Eigenschaften und Verhaltensweisen für Autoobjekte. 
// Bei der Erstellung eines Objekts können wir bestimmte Werte für die Eigenschaften festlegen.